.circleci/config.yml

```diff
@@ -0,0 +1,219 @@
+version: 2
+jobs:
+    build:
+        working_directory: ~/fxa
+        docker:
+        - image: circleci/node:8
+          env:
+          - DISPLAY=:99
+
+        steps:
+        - checkout
+        - run:
+              command: |
+                  sudo apt-get install -y python-setuptools python-dev build-essential xvfb
+                  sudo easy_install pip
+                  sudo pip install mozdownload mozinstall
+                  mozdownload --version 58.0 --destination firefox.tar.bz2
```

Comment:
filed a follow up for caching this: https://github.com/mozilla/fxa-content-server/issues/6414

Reply:
why 58.0?

Reply:
that's what we used in the old circle file